### PR TITLE
Add scenario set and pause state endpoints

### PR DESCRIPTION
**Pull request by Nekroze**
When building mocks for developing a web ui against setting and pausing state to stay in the middle of a scenario would be super handy... so here it is.

Hope this meets all standards, happy to make any improvements first.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,11 +312,26 @@ Like stubbing this call also uses the same DSL to filter and query requests.
 
 ### Scenario
 
-**Title** : Clean all scenarios status.<br>
+**Title** : Clean all scenarios status and pause state.<br>
```
Comment (jmartincmp) [inline README.md]: Did you mean unpaused in this case?

**Comment by jmartin82** [comment]
Happy to see different uses of mmock. I was wondering if anyone uses the scenario feature, haha.

The change seems pretty straightforward and useful.

I'm only waiting for your response to one question in PR to merge it.

File: README.md (hunk 1 above)
Comment (Nekroze) [reply README.md]: I can see the ambiguity, I mean the state of the pause variable/switch. To me it reads as clearing all pause/status state back to default. Should I adjust?

**Comment by Nekroze** [comment]
Love the scenario feature actually, I find it crucial to justifying mmock usage.

Great stuff!

File: README.md (hunk 1 above)
Comment (jmartin82) [reply README.md]: Gotcha! It's ok.

**Comment by Nekroze** [comment]
Thanks. Can this update be pushed to the [Docker hub](https://hub.docker.com/r/jordimartin/mmock) image? Any chance of getting an automated image build?

**Comment by jmartin82** [comment]
It's already there. With the latest tag.

**Comment by Nekroze** [comment]
https://hub.docker.com/r/jordimartin/mmock/tags/ shows the latest tag was last pushed 2 months ago. Have I got the wrong image?

**Comment by jmartin82** [comment]
Fixed! I didn't realize that there was an error pushing the image.

**Comment by Nekroze** [comment]
Awesome!

thanks @jmartin82

**Comment by Nekroze** [comment]
@jmartin82 wanted to share with you how I/we use mmock https://github.com/JumboInteractiveLimited/Gandalf thanks again for a great tool!

**Pull request merged**